back-end/api/controllers/projects.controller.js

```diff
@@ -1,121 +1,123 @@
+import ProjectsService from "../../services/projects.service.js";
 import ProjectsDAO from "../../dao/projectsDAO.js";
-import {project} from "../../entities/project.js";
-
 
 export default class ProjectsController {
-
     /**
-     * A POST API for creating a project object in MongoDB.
-     * @param req
-     * @param res
-     * @param next
-     * @returns {Promise<void>}
-     */
+    * A POST API for creating a project object in MongoDB.
+    * @param req
+    * @param res
+    * @param next
+    * @returns {Promise<void>}
+]   */
     static async apiCreateProject(req, res, next) {
```

Comment:
Nice changes on the controller! We should remove the "static" keyword for each method here since it's unnecessary.